server.py

```diff
@@ -1,37 +1,39 @@
 import json
 import traceback
+import os
+import rethinkdb as r
 
 from flask import Flask, jsonify, render_template, request, make_response
 
 import endpoints
 from utils.ratelimits import ratelimit
 
-app = Flask(__name__, template_folder='views', static_folder='views/assets')
+from dashboard import dash
 
+config = json.load(open('config.json'))
 
-def get_auth_keys():
-    try:
-        with open('keys.json') as keys:
-            data = json.load(keys)
-            if not isinstance(data, list):
-                print('keys.json must only contain an array of valid auth tokens')
-                return []
-            else:
-                return data
-    except FileNotFoundError:
-        print('keys.json wasn\'t found in the current directory')
-        return []
+RDB_ADDRESS = config['rdb_address']
+RDB_PORT = config['rdb_port']
+RDB_DB = config['rdb_db']
+
+rdb = r.connect(RDB_ADDRESS, RDB_PORT)
+
+app = Flask(__name__, template_folder='views', static_folder='views/assets')
+app.register_blueprint(dash)
+
+app.config['SECRET_KEY'] = config['client_secret']
+os.environ['OAUTHLIB_INSECURE_TRANSPORT'] = 'true'
 
 
 def require_authorization(func):
     def wrapper(*args, **kwargs):
-        if request.headers.get('authorization', None) in get_auth_keys():
+        try:
+            list(r.db(RDB_DB).table('keys').filter(r.row['id'] == request.headers.get('authorization', None)).run(rdb))[0]

```

Comment:
`.table('keys').get(request.headers.get('authorization', '')).coerce_to('bool').default(False)`.

This also removes the need for the try/except.